lld/ELF/MarkLive.cpp

```diff
@@ -85,6 +85,12 @@ static uint64_t getAddend(InputSectionBase &sec,
   return rel.r_addend;
 }
 
+template <class ELFT>
```

Comment:
Can we add a comment that we assume all input Crel relocations will have an explicit addend?